disttask/loaddata/scheduler.go

```diff
@@ -101,9 +106,26 @@ func (s *ImportScheduler) SplitSubtask(ctx context.Context, bs []byte) ([]proto.
 }
 
 // OnSubtaskFinished implements the Scheduler.OnSubtaskFinished interface.
-func (s *ImportScheduler) OnSubtaskFinished(context.Context, []byte) error {
+func (s *ImportScheduler) OnSubtaskFinished(ctx context.Context, bs []byte) error {
```

Comment:
what's the meaning of `bs`, it's the message of finishing status? please give a better name